book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
```

Comment:
- [ ] Retels should have `Getters and Setters` since we'll be making modifications by class method